File: AtCoder/abc447_A.py
n, m = map(int, input().split(" "))


if n == m and n != 1:
    print("No")
else:
    if n % 2 == 0 and (n // 2) >= m:
        print("Yes")
    elif n % 2 == 1 and (n // 2) + 1 >= m:
        print("Yes")
    else:
        print("No")

File: AtCoder/abc447_A.cpp
#include<iostream>
using namespace std;

int main(){
    int n, m;
    cin >> n >> m;
    if (n == m && n != ){
    	cout << "No";
    }
    else if (n % 2 == 0 && (n / 2) >= m){
    	cout << "Yes";
    }
    else if (n % 2 == 1 && (n / 2) + 1 >= m){
    	cout << "Yes";
    }
    else{
    	cout << "No";
    }
}